pkg/events/parsing/params.go

```diff
@@ -0,0 +1,85 @@
+package parsing
```

Comment:
I like this as a subpackage!
Here, i'd only like to suggest that you could slightly change the package's name, and then make the exported function names shorter, for example:
`parseargs.Int32Val` or `parse.EventArgInt32Val`, or anything like that. But either way it's a nice change.

Reply:
The convention I know we agreed on is that packages names should be the verb of what they are doing, and the files names should be the noun of the operation (for example, we have the `sorting` package and within the `sorter.go` file which export the a sorter type).
I agree with you that the 'GetEvent' part is redundant, because the package name is events/parsing.

Reply:
Fair enough. Shortening the names is the main benefit either way.

Reply:
I actually want to go back on this. `parse` is also a verb, and, you named the other package `derive` on the same scale, not `deriving`. Let's go for the simple word.